develop/scan-images/index.md

```diff
@@ -4,13 +4,17 @@ description: Scan images best practices guide
 keywords: docker scan, scan, images, snyk, vulnerability
 ---
 
-{% include sign-up-cta.html
```

Comment:
question: are we sure we want to remove the call to action from here ? There isn't another place in this page where we mention hub users have 10 free scans per month.

Reply:
Updated